alibi_detect/utils/pytorch/kernels.py

```diff
@@ -68,16 +68,24 @@ def __init__(
     def sigma(self) -> torch.Tensor:
         return self.log_sigma.exp()
 
-    def forward(self, x: Union[np.ndarray, torch.Tensor], y: Union[np.ndarray, torch.Tensor],
-                infer_sigma: bool = False) -> torch.Tensor:
+    def forward(self, x: Union[np.ndarray, torch.Tensor],
```

Comment:
Nitpicking big time here, but let's try to keep same type of indentation as e.g. in the `DeepKernel` below.